### ISSUE TITLE
Prod Firewall - app to DDW database

### ISSUE THREAD

**Issue by nathankota** [uic-utah/uic-inventory#313]
[INC1976688](https://utah.service-now.com/now/nav/ui/classic/params/target/incident.do%3Fsys_id%3D8e2665efc386861475b66b159901314f%26sysparm_stack%3D%26sysparm_view%3D)

[REQ0192349](https://utah.service-now.com/sc_request.do?sys_id=08c417d79739ce5052a6bdb8c253af02&sysparm_record_target=sc_request&sysparm_catalog=e0d08b13c3330100c8b837659bba8fb4&sysparm_catalog_view=catalog_Service_Catalog)
[RITM0270625](https://utah.service-now.com/sc_req_item.do?sys_id=8cc417d79739ce5052a6bdb8c253af02&sysparm_record_target=sc_req_item&sysparm_record_row=1&sysparm_record_rows=1&sysparm_record_list=request%3D08c417d79739ce5052a6bdb8c253af02%5EORDERBYDESCsys_created_on)

ref: #229 for [RITM0237929](https://utah.service-now.com/sc_req_item.do?sys_id=7271bd8a475225d4184a874c346d43e3&sysparm_record_target=sc_req_item&sysparm_record_row=1&sysparm_record_rows=1&sysparm_record_list=request%3Dfa71bd8a475225d4184a874c346d43e2%5EORDERBYDESCsys_created_on) containing the destination information

**Comment by nathankota** [comment]
First round of testing failed. Firewall rejected.

Dev


![Image](https://github.com/uic-utah/uic-inventory/assets/15659642/7103229f-e65b-4da0-a2d3-74fb9aaa4603)


Prod



![Image](https://github.com/uic-utah/uic-inventory/assets/15659642/7b0dc5e6-2008-4503-95eb-f49b181e0a04)

Log


![Image](https://github.com/uic-utah/uic-inventory/assets/15659642/4f68f440-db39-4c10-9c53-82dff4d918da)

**Comment by rsparker-utah** [comment]
Validated. I tested this in the prod environment by creating a new site and submitting a well inventory in an area that I knew was in a groundwater protection zone. The water system information was displaying as expected within the "Location Details" section of the inventory report:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/5ed68db8-7038-4912-a954-f6e428e42bd7)
I also verified that this information was showing up in the subsequent Inventory Report Packet for this site:
![image](https://github.com/uic-utah/uic-inventory/assets/49735267/0d95f663-96f1-44db-9e6e-f48eb2d41a48)
As an extra check, I cross referenced the water system information with the Division of Drinking Water's [Waterlink](https://waterlink.utah.gov/reports.html?systemId=1306) Database to confirm the information was displaying correctly.
